### ISSUE TITLE
Change bold tags that are used for formatting

### ISSUE THREAD

**Issue by HaSistrunk** [RockefellerArchiveCenter/aurora#265]
Where bold text is only used for formatting, use css or headings instead.
- Organization Profile page in Organization box (including "edit organization" button)
- User Profile page in admin box including the "edit user" button
-  Browseable API
- BagIt Profile view page

**Comment by helrond** [comment]
Does this apply to `<strong>` tags as well, or just `<b>` tags?

**Comment by HaSistrunk** [comment]
Good question. If the text should be emphasized semantically for some reason, `<strong>` is fine. If `<strong>` is just being used for visual effect, css should be used. We can assess how this bold text is functioning on the page, but my sense was that it is bold for aesthetic purposes rather than to emphasize the text.
